File: test/resources/repos/csharp/test_repo/Models/Person.cs
using TestProject;

namespace TestProject.Models
{
    public class Person
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public string Email { get; set; }
        
        public Person(string name, int age, string email)
        {
            Name = name;
            Age = age;
            Email = email;
        }
        
        public override string ToString()
        {
            return $"{Name} ({Age}) - {Email}";
        }
        
        public bool IsAdult()
        {
            return Age >= 18;
        }
        
        public int CalculateYearsUntilRetirement()
        {
            var calculator = new Calculator();
            return calculator.Subtract(65, Age);
        }

        public void CalculateAge()
        {
            var calc = new Calculator();
            var result = calc.Subtract(2025, 1990); // This creates the reference the test needs
        }
}

File: test/resources/repos/vbnet/test_repo/Models/Person.vb
Imports TestProject

Namespace TestProject.Models
    Public Class Person
        Public Property Name As String
        Public Property Age As Integer
        Public Property Email As String

        Public Sub New(name As String, age As Integer, email As String)
            Me.Name = name
            Me.Age = age
            Me.Email = email
        End Sub

        Public Overrides Function ToString() As String
            Return $"{Name} ({Age}) - {Email}"
        End Function

        Public Function IsAdult() As Boolean
            Return Age >= 18
        End Function

        Public Function CalculateYearsUntilRetirement() As Integer
            Dim calculator As New Calculator()
            Return calculator.Subtract(65, Age)
        End Function
    End Class
End Namespace
